### ISSUE TITLE
Can't reproduce the accuracy from wrn_40_2 to wrn_16_2

### ISSUE THREAD

**Issue by shicaiwei123** [megvii-research/mdistiller#8]
Hello, I  have reproduced the experimental results on the imagenet and some results on cifra100, such as, from VGG13 to MobileNet-V2. 

However, the best result from wrn_40_2 to the wrn_16_2 is only 75.15, which is significantly smaller than the 76.24 in the paper (Tabel 11).

All the hyper-parameters used are from the config folder of this repository. And the platform is 1080ti on ubuntu 18.04. 

Is there anything I have overlooked? 

Or could you share the log for this experiment?

**Comment by Zzzzz1** [comment]
Details about our experiment environment can be seen at #issue7.

**Comment by xiaohe725** [comment]
> Hello, I have reproduced the experimental results on the imagenet and some results on cifra100, such as, from VGG13 to MobileNet-V2.
> 
> However, the best result from wrn_40_2 to the wrn_16_2 is only 75.15, which is significantly smaller than the 76.24 in the paper (Tabel 11).
> 
> All the hyper-parameters used are from the config folder of this repository. And the platform is 1080ti on ubuntu 18.04.
> 
> Is there anything I have overlooked?
> 
> Or could you share the log for this experiment?

Hello, have you solved this problem? I have also encountered the same problem

**Comment by helloworld77** [comment]
hello, wrn40_2_wrn_16_2.yaml avg acc is 75.505, for one gpu train 4 times.

**Comment by helloworld77** [comment]
In the paper, the teacher wrn40_2 is only 75.61, but dkd could make wrn-16-2 to  76.24 acc.

**Comment by wutaiqiang** [comment]
> In the paper, the teacher wrn40_2 is only 75.61, but dkd could make wrn-16-2 to 76.24 acc.

similar results that much lower than reported in paper.
